app/javascript/stylesheets/workTypes.scss

```diff
@@ -0,0 +1,41 @@
+.work-types {
+  a {
+    text-decoration: none;
+
+    &:after {
+      content: '\f078';
+      font-family: Fontawesome;
+      padding-left: 0.5rem;
+    }
+
+    &.collapsed:after {
+      content: '\f054';
+    }
+  }
+
+  h6 {
+    color: $stanford-digital-red;
+    font-weight: bold;
+  }
+
+  input[type="radio"].toggle + label {
+    font-size: smaller;
+    line-height: 1;
+    padding: 0.375rem 0.375rem;
+  }
+}
+
+.work-type-box {
+  @extend .col-md-3;
+  @extend .mb-5;
+  @extend .mx-4;
```

Comment:
This is the line that makes the work type boxes render three-per-row rather than four-per-row.